cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -1706,11 +1706,13 @@ func TestParsePluginConfig(t *testing.T) {
 		name                    string
 		pluginYAMLConf          string
 		exp_versions_in_summary VersionsInSummary
+		exp_status_code         codes.Code
```

Comment:
So this will be `expectedError   error` and the check can use `errors.Is(err, tc.expectedError)`. You'll need to check what kind of error is returned for `json.Unmarshal` in that case etc..

Reply:
Added two new error codes.